css/style.css

```diff
@@ -1,19 +1,253 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  margin: 0;
+  color: var(--white);
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+h1 {
+  margin: 0;
+}
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+:root {
+  --orange-dark: #f15b2a;
+  --white: #fff;
+  --grey-light: #959697;
+  --grey-dark: #6b7278;
+}
+
+/* Header */
+
+.header {
+  padding: 0 5rem;
+}
+
+.header {
+  display: flex;
+  background-color: var(---white);
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header__logo {
+  max-height: 50px;
+  width: 125px;
+}
+
+/* Navigation */
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  padding: 0.5rem 0;
+}
+
+.nav__link {
+  color: var(--grey-light);
+  font-weight: 400;
+  text-decoration: none;
+  padding: 1rem;
+}
+
+li:first-child a {
+  font-weight: 600;
+}
+
+nav a:hover {
+  color: var(--orange-dark);
+}
+
+.nav__mobile {
+  display: none;
+}
+
+/* Main */
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.main {
+  background-image: url("../img/first-background.jpg");
+  background-size: cover;
+  background-position: center;
+  height: 100vh;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.main h1 {
+  font-size: 72px;
+  font-weight: 300;
+}
+
+h2 {
+  font-size: 40px;
+  margin-bottom: 4rem;
+  font-weight: 100;
+}
+
+.btn {
+  color: var(--white);
+  background-color: var(--orange-dark);
+  border-radius: 3px;
+  border: none;
+  padding: 10px 24px;
+}
+
+.btn:hover {
+  color: var(--orange-dark);
+  background-color: var(--white);
+}
+
+/* Service section Under Main Image */
+
+.service__section {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 2rem;
+}
+
+.service__thumbnail {
+  max-height: 100px;
+}
+
+.service {
+  text-align: center;
+  padding: 30px 60px;
+}
+
+.heading__underMainImage,
+.service__summary {
+  color: var(--grey-dark);
+  text-align: center;
+}
+
+.heading__underMainImage {
+  margin-top: 5rem;
+  margin-bottom: 1rem;
+}
+
+/* Advert section */
+
+.advert__section {
```

Comment:
If you add `overflow: hidden;` you'll be able to see how the browser will not longer overflow blank space on the right hand side (you shouldn't be able to scroll left and right, it's a bad user experience). 

Add that to see what it should feel like, then play around with the child components to get the two items centered.